spec/services/article_api_index_service_spec.rb

```diff
@@ -0,0 +1,36 @@
+require "rails_helper"
```

Comment:
I think we'd better transfer these tests into https://github.com/forem/forem/blob/master/spec/requests/api/v0/articles_spec.rb where all the others are instead of testing the service directly. It's not wrong, it's just that all those API tests are integration tests

Reply:
I was a little surprised that there are no service tests yet. Anyway, I moved them, as you suggested.

Reply:
@rafaltrojanowski There are quite a few specs for services in `spec/services`:

```
$ ls -1 spec/services/**/*.rb | wc -l
     115
```